Grammar fix / improvements (#4)
Lautenschlager-id/Transfromage: docs/Events.md

>				timestamp = 0, -- When the topic was created.
>				author = "", -- The name of the topic author.
>				content = "", -- The content of the message.

> >				canLike = false, -- Whether the bot can like/dislike a message.

Reviewer: ```suggestion
>				canLike = false, -- Whether the account can like/dislike the message.
```

>			timestamp = 0, -- When the topic was created.
>			author = "", -- The name of the topic author.
>			content = "", -- The content of the message.

> >			canLike = false, -- Whether the bot can like/dislike a message.

Reviewer: ```suggestion
>			canLike = false, -- Whether the account can like/dislike the message.
```

>	timestamp = 0, -- When the topic was created.
>	author = "", -- The name of the topic author.
>	content = "", -- The content of the message.

> >	canLike = false, -- Whether the bot can like/dislike a message.

Reviewer: ```suggestion
>	canLike = false, -- Whether the account can like/dislike the message.
```

>			timestamp = 0, -- When the topic was created.
>			author = "", -- The name of the topic author.
>			content = "", -- The content of the message.

> >			canLike = false, -- Whether the bot can like/dislike a message.

Reviewer: ```suggestion
>			canLike = false, -- Whether the account can like/dislike the message.
```

>| :-: | :-: | - |
>| tribeName | `string` | The name of the tribe. |
>

> >Triggered when a player joins a tribe house.

Reviewer: ```suggestion
>Triggered when the account joins a tribe house.
```

>
>
>Triggered when a player joins the room.
>
>**@playerData structure**:
>```Lua
>{
>	playerName = "", -- The nickname of the player.
>	id = 0, -- The temporary id of the player during the section.
>	isShaman = false, -- Whether the player is shaman or not.
>	isDead = false, -- Whether the player is dead or alive.
>	score = 0, -- The current player's score.
>	hasCheese = false, -- Whether the player has cheese or not.
>	title = 0, -- The player's title id.
>	titleStars = 0, -- The number of stars the player's title has.
>	gender = 0, -- The player's gender. Enum in enum.gender.
>	look = "", -- The current outfit string code of the player.
>	mouseColor = 0, -- The color of the player. It is set to -1 if it's the default color.
>	shamanColor = 0, -- The color of the player as shaman.
>	nameColor = 0, -- The color of the nickname of the player.
>	isSouris = false, -- Whether the player is souris or not.
>	isVampire = false, -- Whether the player is vampire or not.
>	hasWon = false, -- Whether the player has entered the hole in the round or not.
>	winPosition = 0, -- The position where the player entered the hole. It is set to -1 if it has not won yet.

> >	winTimeElapsed = 0, -- The time elapsed till the player enters the hole. It is set to -1 if it has not won yet.

Reviewer: Maybe
```suggestion
>	winTimeElapsed = 0, -- The current time of when the player entered the hole. It is set to -1 if it has not won yet.
```
?

>### newTribeMember ( memberName )
>| Parameter | Type | Description |
>| :-: | :-: | - |

> >| memberName | `string` | The name of the new tribeMember. |

Reviewer: ```suggestion
>| memberName | `string` | The name of the new tribe member. |
```

>	playerName = "", -- The nickname of the player.
>	id = 0, -- The temporary id of the player during the section.
>	isShaman = false, -- Whether the player is shaman or not.
>	isDead = false, -- Whether the player is dead or alive.
>	score = 0, -- The current player's score.
>	hasCheese = false, -- Whether the player has cheese or not.
>	title = 0, -- The player's title id.
>	titleStars = 0, -- The number of stars the player's title has.
>	gender = 0, -- The player's gender. Enum in enum.gender.
>	look = "", -- The current outfit string code of the player.
>	mouseColor = 0, -- The color of the player. It is set to -1 if it's the default color.
>	shamanColor = 0, -- The color of the player as shaman.
>	nameColor = 0, -- The color of the nickname of the player.
>	isSouris = false, -- Whether the player is souris or not.
>	isVampire = false, -- Whether the player is vampire or not.
>	hasWon = false, -- Whether the player has entered the hole in the round or not.
>	winPosition = 0, -- The position where the player entered the hole. It is set to -1 if it has not won yet.

> >	winTimeElapsed = 0, -- The time elapsed till the player enters the hole. It is set to -1 if it has not won yet.

Reviewer: ```suggestion
>	winTimeElapsed = 0, -- The current time of when the player entered the hole. It is set to -1 if it has not won yet.
```

>
>
>Triggered when a player gets (or loses) the cheese.
>
>**@playerData structure**:
>```Lua
>{
>	playerName = "", -- The nickname of the player.
>	id = 0, -- The temporary id of the player during the section.
>	isShaman = false, -- Whether the player is shaman or not.
>	isDead = false, -- Whether the player is dead or alive.
>	score = 0, -- The current player's score.
>	hasCheese = false, -- Whether the player has cheese or not.
>	title = 0, -- The player's title id.
>	titleStars = 0, -- The number of stars the player's title has.
>	gender = 0, -- The player's gender. Enum in enum.gender.
>	look = "", -- The current outfit string code of the player.
>	mouseColor = 0, -- The color of the player. It is set to -1 if it's the default color.
>	shamanColor = 0, -- The color of the player as shaman.
>	nameColor = 0, -- The color of the nickname of the player.
>	isSouris = false, -- Whether the player is souris or not.
>	isVampire = false, -- Whether the player is vampire or not.
>	hasWon = false, -- Whether the player has entered the hole in the round or not.
>	winPosition = 0, -- The position where the player entered the hole. It is set to -1 if it has not won yet.

> >	winTimeElapsed = 0, -- The time elapsed till the player enters the hole. It is set to -1 if it has not won yet.

Reviewer: ```suggestion
>	winTimeElapsed = 0, -- The current time of when the player entered the hole. It is set to -1 if it has not won yet.
```

>**@playerData structure**:
>```Lua
>{
>>	playerName = "", -- The nickname of the player.
>	id = 0, -- The temporary id of the player during the section.
>	isShaman = false, -- Whether the player is shaman or not.
>	isDead = false, -- Whether the player is dead or alive.
>	score = 0, -- The current player's score.
>	hasCheese = false, -- Whether the player has cheese or not.
>	title = 0, -- The player's title id.
>	titleStars = 0, -- The number of stars the player's title has.
>	gender = 0, -- The player's gender. Enum in enum.gender.
>	look = "", -- The current outfit string code of the player.
>	mouseColor = 0, -- The color of the player. It is set to -1 if it's the default color.
>	shamanColor = 0, -- The color of the player as shaman.
>	nameColor = 0, -- The color of the nickname of the player.
>	isSouris = false, -- Whether the player is souris or not.
>	isVampire = false, -- Whether the player is vampire or not.
>	hasWon = false, -- Whether the player has entered the hole in the round or not.
>	winPosition = 0, -- The position where the player entered the hole. It is set to -1 if it has not won yet.

> >	winTimeElapsed = 0, -- The time elapsed till the player enters the hole. It is set to -1 if it has not won yet.

Reviewer: ```suggestion
>	winTimeElapsed = 0, -- The current time of when the player entered the hole. It is set to -1 if it has not won yet.
```

>**@playerData structure**:
>```Lua
>{
>>	playerName = "", -- The nickname of the player.
>	id = 0, -- The temporary id of the player during the section.
>	isShaman = false, -- Whether the player is shaman or not.
>	isDead = false, -- Whether the player is dead or alive.
>	score = 0, -- The current player's score.
>	hasCheese = false, -- Whether the player has cheese or not.
>	title = 0, -- The player's title id.
>	titleStars = 0, -- The number of stars the player's title has.
>	gender = 0, -- The player's gender. Enum in enum.gender.
>	look = "", -- The current outfit string code of the player.
>	mouseColor = 0, -- The color of the player. It is set to -1 if it's the default color.
>	shamanColor = 0, -- The color of the player as shaman.
>	nameColor = 0, -- The color of the nickname of the player.
>	isSouris = false, -- Whether the player is souris or not.
>	isVampire = false, -- Whether the player is vampire or not.
>	hasWon = false, -- Whether the player has entered the hole in the round or not.
>	winPosition = 0, -- The position where the player entered the hole. It is set to -1 if it has not won yet.

> >	winTimeElapsed = 0, -- The time elapsed till the player enters the hole. It is set to -1 if it has not won yet.

Reviewer: ```suggestion
>	winTimeElapsed = 0, -- The current time of when the player entered the hole. It is set to -1 if it has not won yet.
```

>| Parameter | Type | Description |
>| :-: | :-: | - |
>| playerData | `table` | The data of the player. |
>| position | `int` | The position where the player entered the hole. |

> >| timeElapsed | `number` | The time elapsed until the player entered the hole. |

Reviewer: ```suggestion
>| timeElapsed | `number` | The current time of when the player entered the hole. |
```

>
>
>Triggered when a player joins the hole.
>
>**@playerData structure**:
>```Lua
>{

> >>	playerName = "", -- The nickname of the player.

Reviewer: Double `>` ?

>	id = 0, -- The temporary id of the player during the section.
>	isShaman = false, -- Whether the player is shaman or not.
>	isDead = false, -- Whether the player is dead or alive.
>	score = 0, -- The current player's score.
>	hasCheese = false, -- Whether the player has cheese or not.
>	title = 0, -- The player's title id.
>	titleStars = 0, -- The number of stars the player's title has.
>	gender = 0, -- The player's gender. Enum in enum.gender.
>	look = "", -- The current outfit string code of the player.
>	mouseColor = 0, -- The color of the player. It is set to -1 if it's the default color.
>	shamanColor = 0, -- The color of the player as shaman.
>	nameColor = 0, -- The color of the nickname of the player.
>	isSouris = false, -- Whether the player is souris or not.
>	isVampire = false, -- Whether the player is vampire or not.
>	hasWon = false, -- Whether the player has entered the hole in the round or not.
>	winPosition = 0, -- The position where the player entered the hole. It is set to -1 if it has not won yet.

> >	winTimeElapsed = 0, -- The time elapsed till the player enters the hole. It is set to -1 if it has not won yet.

Reviewer: ```suggestion
>	winTimeElapsed = 0, -- The current time of when the player entered the hole. It is set to -1 if it has not won yet.
```

>| playerList | `table` | The data of all players. |
>
>

> >Triggered when the data of all players are refreshed (mostly when a new map is loadedgames).

Reviewer: ```suggestion
>Triggered when the data of all players are refreshed (mostly when a new map is loaded).
```

>		playerName = "", -- The nickname of the player.
>		id = 0, -- The temporary id of the player during the section.
>		isShaman = false, -- Whether the player is shaman or not.
>		isDead = false, -- Whether the player is dead or alive.
>		score = 0, -- The current player's score.
>		hasCheese = false, -- Whether the player has cheese or not.
>		title = 0, -- The player's title id.
>		titleStars = 0, -- The number of stars the player's title has.
>		gender = 0, -- The player's gender. Enum in enum.gender.
>		look = "", -- The current outfit string code of the player.
>		mouseColor = 0, -- The color of the player. It is set to -1 if it's the default color.
>		shamanColor = 0, -- The color of the player as shaman.
>		nameColor = 0, -- The color of the nickname of the player.
>		isSouris = false, -- Whether the player is souris or not.
>		isVampire = false, -- Whether the player is vampire or not.
>		hasWon = false, -- Whether the player has entered the hole in the round or not.
>		winPosition = 0, -- The position where the player entered the hole. It is set to -1 if it has not won yet.

> >		winTimeElapsed = 0, -- The time elapsed till the player enters the hole. It is set to -1 if it has not won yet.

Reviewer: ```suggestion
>		winTimeElapsed = 0, -- The current time of when the player entered the hole. It is set to -1 if it has not won yet.
```

>			timestamp = 0, -- When the topic was created.
>			author = "", -- The name of the topic author.
>			content = "", -- The content of the message.

> 			canLike = false, -- Whether the bot can like/dislike a message.

Reviewer: ```suggestion
			canLike = false, -- Whether the account can like/dislike the message.
```

>	playerName = "", -- The nickname of the player.
>	id = 0, -- The temporary id of the player during the section.
>	isShaman = false, -- Whether the player is shaman or not.
>	isDead = false, -- Whether the player is dead or alive.
>	score = 0, -- The current player's score.
>	hasCheese = false, -- Whether the player has cheese or not.
>	title = 0, -- The player's title id.
>	titleStars = 0, -- The number of stars the player's title has.
>	gender = 0, -- The player's gender. Enum in enum.gender.
>	look = "", -- The current outfit string code of the player.
>	mouseColor = 0, -- The color of the player. It is set to -1 if it's the default color.
>	shamanColor = 0, -- The color of the player as shaman.
>	nameColor = 0, -- The color of the nickname of the player.
>	isSouris = false, -- Whether the player is souris or not.
>	isVampire = false, -- Whether the player is vampire or not.
>	hasWon = false, -- Whether the player has entered the hole in the round or not.
>	winPosition = 0, -- The position where the player entered the hole. It is set to -1 if it has not won yet.

> >	winTimeElapsed = 0, -- The time elapsed till the player enters the hole. It is set to -1 if it has not won yet.

Reviewer: ```suggestion
>	winTimeElapsed = 0, -- The current time of when the player entered the hole. It is set to -1 if it has not won yet.
```
